tests/specs/lint/lint_plugin_no_fixer/lint.out

```diff
@@ -0,0 +1,2 @@
+[WILDCARD]Found 1 problem
+Checked 1 file
```

Comment:
It might be better to test the full output so that the test is more resilient to change (ex. the fixes being moved somewhere else)